proto/gateway.proto

```diff
@@ -52,6 +52,7 @@ message create_bdev_req {
 	string rbd_pool_name = 2;
 	string rbd_image_name = 3;
 	int32 block_size = 4;
+	optional string uuid = 5;
```

Comment:
Shouldn't we update the CLI component to expose this option too?

Reply:
@epuertat , we didn't want to expose the UUID in the CLI. We wanted a value to be automatically created and then become persistent. Kept in the OMAP file. So, if the gateway restarts the UUID will be restored.

Reply:
I don't see a use case in which the user will want to set the UUID himself via a CLI. Other opinions?